lkp-exec/install

```diff
@@ -350,6 +350,10 @@ install_base_support()
 		(
 			cd $LKP_SRC || exit
 			gem install bundler -v $bundler_version
+			# Adds bundler to PATH for OpenSUSE
+			if [ "$distro" = "opensuse" ]; then
+				export PATH=$PATH:/usr/lib64/ruby/gems/2.5.0/gems/bundler-$bundler_version/exe
```

Comment:
thanks, is it possible to use $(gem --version) here for 2.5.0 to make it more general?

Reply:
Ah that's a great suggestion! Will change it to make it more general. 😄